server/middleware/classrooms.coffee

```diff
@@ -151,7 +162,7 @@ module.exports =
     code = req.body.code.toLowerCase()
     classroom = yield Classroom.findOne({code: code})
     if not classroom
-      throw new errors.NotFound(res) 
+      throw new errors.NotFound(res)
```

Comment:
Ah, oops! Yeah, that should be 'Classroom not found.' or something along those lines. Guess I didn't refactor it quite correctly...